### PR TITLE
Return User to the same page where they were after registration

### PR DESCRIPTION
**Pull request by grebiniuk**
## Description
Added storing return path in session storage so the user will be returned to the same page after registration and press on the button "Continue shopping".

#### Screenshots
image

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/containers/header-profile/header-profile.js [hunk 1]
```diff
@@ -89,7 +93,14 @@ const HeaderProfile = ({ fromSideBar }) => {
     {
       value: PROFILE_OPTIONS_VALUES[language].logIn,
       icon: <ExitToAppIcon />,
-      clickHandler: () => handleRedirect('/login')
+      clickHandler: () => {
+        const pathName = history.location.pathname;
+        const returnPath =
+          (pathName === pathToRegister || pathName === pathToLogin ? '/' : pathName) +
```
Comment (yuliiakucher) [inline src/containers/header-profile/header-profile.js:99]: please remove magic string and use const pathToMain from configs

File: src/containers/header-profile/header-profile.js [hunk 2]
```diff
@@ -89,7 +93,14 @@ const HeaderProfile = ({ fromSideBar }) => {
...
+          history.location.search;
+        sessionStorage.setItem(RETURN_PAGE, returnPath);
+        handleRedirect('/login');
```
Comment (yuliiakucher) [inline src/containers/header-profile/header-profile.js:102]: please remove magic string and use const pathToLogin from configs

File: src/containers/header-profile/header-profile.js (hunk 1 above)
Comment (grebiniuk) [reply src/containers/header-profile/header-profile.js]: Fixed, thank you

File: src/containers/header-profile/header-profile.js (hunk 2 above)
Comment (grebiniuk) [reply src/containers/header-profile/header-profile.js]: Fixed, Thank you

**Review by yuliiakucher — APPROVED** [review]

**Review by twihell — APPROVED** [review]

**Pull request merged**